src/client/stores/CreateBountyFormStore.js

```diff
@@ -0,0 +1,16 @@
+import { observable, action } from 'mobx'
+
+class CreateBountyFormStore {
```

Comment:
Does this hold one bounty form, or many?  I feel like calling it a "store" implies many.

Reply:
@jpkcambridge it holds the state for the form view